src/WaveFunction/WaveFunction.cpp

```diff
@@ -62,4 +63,188 @@ double WaveFunction::calculateShannonEntropy() const {
 }
 
 
+/**
+ *  Transform the underlying ONV basis of the wave function (only for FCI [ProductFockSpace]) as shown in Helgaker 11.9
```

Comment:
The documentation in source and header seems out of sync.

```suggestion
 *  Transform the underlying ONV basis of the wave function (only for FCI [ProductFockSpace]), using the algorithm as in Helgaker chapter 11.9
```